### DISCUSSION TITLE
~/.local/bin/ in the $PATH

### DISCUSSION THREAD

**Discussion by se7en-x230** [prasanthrangan/hyprdots#762] [General]
New install. 
scripts launch without problem from the term

But it is not included in rofi 
and will not be started with a keybinding. 
bind = $mainMod, M, exec, myscript 
does not work while
$ myscript works

Sorry my fault

**Comment by kRHYME7** [comment]
>But it is not included in rofi
and will not be started with a keybinding.

Hello! Can you elaborate more please? 
Do you mean your scripts in ``` ~/.local/bin/  ``` won't run if added as keybinding? 

what might be the output for your ``` echo $PATH ``` ? 

which file did you add the keybinds?
